components/demo/demo-snippet.js

```diff
@@ -40,15 +43,19 @@ class DemoSnippet extends LitElement {
 			}
 			.d2l-demo-snippet-demo-wrapper.fullscreen {
 				background-color: white;
+				height: fit-content;
 				inset: 0;
 				overflow: auto;
-				position: fixed;
+				position: absolute;
```

Comment:
This pushes the scrolling back into the body box model or whatever, which allows Polypane to sync scrolling in fullscreen mode.

![scrolling-sync](https://user-images.githubusercontent.com/1289042/202264038-cbeebdf5-dbdd-409e-8ada-25ca8cf93fa1.gif)